electron/games.ts

```diff
@@ -298,6 +299,7 @@ class LegendaryGame implements Game {
       audio: audioFix ? `PULSE_LATENCY_MSEC=60` : '',
       fps: showFps ? `DXVK_HUD=fps` : '',
       other: otherOptions ? otherOptions : '',
+      prime: nvidiaPrime ? '__NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia' : '',
```

Comment:
I think there was another related issue where different environment variables had to be used. Not sure though.

Reply:
True. But in this case we will use the same behavior as lutris since I think they know best. 👍